packages/node/examples/express/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start:metrics": "DEBUG=express:router npx node-dev index.js",
-    "start:webhooks": "DEBUG=express:router npx node-dev webhook.js"
+    "start:webhooks": "DEBUG=express:router npx node-dev index.js"
```

Comment:
I originally thought it was a good idea to have them as separate apps, but with other languages not being quite as lightweight as node, I'm happy for us to be consistent and include them in one.